server/src/main/java/com/linecorp/centraldogma/server/internal/api/RepositoryServiceV1.java

```diff
@@ -68,6 +72,7 @@ public RepositoryServiceV1(ProjectManager projectManager, CommandExecutor execut
      * <p>Returns the list of the repositories or removed repositories.
      */
     @Get("/projects/{projectName}/repos")
+    @Decorator(ProjectMembersOnly.class)
```

Comment:
I'm curious if we should allow the listing of the repositories and projects for all non-anonymous (i.e. logged-in) users. (but I believe we should not allow file-listing.)